py/tests/scale.py

```diff
@@ -3,50 +3,80 @@
 import pykms
 import time
 import random
+import argparse
+
+def plane_commit(card, crtc, plane, fb, x, y, w, h) :
+	req = pykms.AtomicReq(card)
+	req.add_plane(plane, fb, crtc, None, (x, y, w, h))
+	r = req.commit_sync()
+	assert r == 0, "Plane commit failed: %d" % r
+
+
+parser = argparse.ArgumentParser(description='Simple scaling stress test.')
+parser.add_argument('--plane', '-p', dest='plane', default="",
+		    required=False, help='plane number to use')
```

Comment:
You want an int from this parameter, but default is string... You can define the type as int, instead. But it can be much simpler:

`parser.add_argument("-p", "--plane", default="")`

should do it just as well (presuming you accept a string). Even the default could perhaps be dropped, as if you handle args.plane as you do below, you can just check if the option was given at all. You don't need "" default, unless you pass that value forward directly without checks.

I don't think help is necessary, especially if we accept similar plane argument in almost all of the tests and utils.